packages/autoloader/src/autoload.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+// autoload_wordpress.php generated by automattic/autoloader
```

Comment:
Can't find `autoload_wordpress.php` anywhere - is that a typo?

Reply:
yes. :(

Reply:
done